flash/vision/classification/data.py

```diff
@@ -231,6 +258,14 @@ def __init__(
         if self._predict_ds:
             self.set_dataset_attribute(self._predict_ds, 'num_classes', self.num_classes)
 
+    @staticmethod
+    def configure_data_fetcher(*args, **kwargs) -> BaseDataFetcher:
+        return _MatplotlibVisualization(*args, **kwargs)
+
+    def show(self) -> None:
```

Comment:
Let's find another way to do that. This is confusing.

Reply:
we can have a better naming here. An alternative but a bit more hacky, is to pass `blocking` flag across all the different functions until the matplotlib `plt.show` calls.